conf/core/ConfigOptions.asciidoc

```diff
@@ -1645,8 +1645,16 @@ A list of map operations to be applied to a map for cleaning purposes, in order.
 [[match.creators]]
 === match.creators
 
-* Data Type: string
-* Default Value: `hoot::BuildingMatchCreator;hoot::ScriptMatchCreator,PoiGeneric.js;hoot::HighwayMatchCreator;hoot::ScriptMatchCreator,LinearWaterway.js;hoot::PoiPolygonMatchCreator;hoot::ScriptMatchCreator,Area.js;hoot::ScriptMatchCreator,Railway.js;hoot::ScriptMatchCreator,PowerLine.js`
+* Data Type: list
+* Default Value:
+** `hoot::BuildingMatchCreator`
+** `hoot::ScriptMatchCreator,PoiGeneric.js`
+** `hoot::HighwayMatchCreator`
+** `hoot::ScriptMatchCreator,LinearWaterway.js`
+** `hoot::PoiPolygonMatchCreator`
+** `hoot::ScriptMatchCreator,Area.js`
+** `hoot::ScriptMatchCreator,Railway.js`
+** `hoot::ScriptMatchCreator,PowerLine.js`
```

Comment:
Pretty simple, just changed this into a list and update calls to get this list, which already split on the `;` and made a list anyway.